lib/chef/knife/vsphere_vm_snapshot.rb

```diff
@@ -19,6 +19,10 @@ class Chef::Knife::VsphereVmSnapshot < Chef::Knife::BaseVsphereCommand
          :long => "--list",
          :description => "The current tree of snapshots"
 
+  option :find,
+         :long => "--find",
+         :description => "Find snapshots for VM name"
```

Comment:
I think this description is at odds with the description on line 83. Could I suggest "find virtual machine instead of specifying a folder"?